src/packets.c

```diff
@@ -529,7 +529,7 @@ TbBool process_dungeon_power_hand_state(long plyr_idx)
       }
       long i = thing_is_creature_special_digger(thing);
       if (can_drop_thing_here(stl_x, stl_y, player->id_number, i)
-        || !can_dig_here(stl_x, stl_y, player->id_number))
+        || !can_dig_here(stl_x, stl_y, player->id_number, false))
```

Comment:
setting `enemy_wall_undiggable` to true here means that if you are carrying a creature, it shows this type of cursor over enemy walls:
![image](https://user-images.githubusercontent.com/1984342/113593794-bd171280-962e-11eb-857d-836ae1786717.png)
rather than a red boundbox and a dig cursor (if it stays set to false)

I suggest this remains as false.